saleor/graphql/order/dataloaders.py

```diff
@@ -234,3 +234,18 @@ def batch_load(self, keys):
         for transaction in transactions:
             transactions_map[transaction.order_id].append(transaction)
         return [transactions_map.get(order_id, []) for order_id in keys]
+
+
+class TransactionEventByOrderGrantedRefundIdLoader(DataLoader):
```

Comment:
```suggestion
class TransactionEventsByOrderGrantedRefundIdLoader(DataLoader):
```
as the list of instances is returned for single id